builder/builder_utilities.livecodescript

```diff
@@ -1,7 +1,7 @@
 ﻿script "BuilderUtilities"
 
 constant kMergExtVersion = "2017-11-3"
-constant kTSNetVersion = "1.3.2"
+constant kTSNetVersion = "1.3.3”
 
```

Comment:
@techstrategies the closing quote is a curly one, could you replace it with a regular one? :)

Reply:
@livecodepanos sorry about that, fixed now! :)

Reply:
Thanks @techstrategies